metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "stahnma-epel",
-  "version": "2.0.0",
-  "author": "stahnma",
+  "name": "puppet-epel",
+  "version": "3.0.0-rc0",
```

Comment:
are you already sure that the next release is a major one? Will we rip out RHEL 5 support?

Reply:
Maybe not, but we won't support puppet 3 anymore for sure.